src/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -293,12 +290,12 @@
     <value>Can not access a closed Stream.</value>
   </data>
   <data name="net_http_libcurl_callback_notsupported" xml:space="preserve">
-    <value>The handler does not support custom handling of certificates with this combination of libcurl ({0}) and its SSL backend ("{1}").</value>
+    <value>The handler does not support custom validation of certificates with this combination of libcurl ({0}) and its SSL backend ("{1}"). A libcurl built with OpenSSL is required.</value>
```

Comment:
When I made these three messages I was very much explicitly trying to avoid saying OpenSSL.  I don't quite remember why, but that was my goal.